src/main/scala/beam/agentsim/agents/PersonAgent.scala

```diff
@@ -825,6 +836,17 @@ class PersonAgent(
       }
   }
 
+  def replanningReasonBuilder(data: BasePersonData, prefix: String): String = {
```

Comment:
The name doesn't give the full definition. This builds AND outputs, so I would suggest something more like 

```suggestion
  def getReplanningReasonFrom(data: BasePersonData, prefix: String): String = {
```

Reply:
method renaming is done